### PR TITLE
MM-1711 update sign-up and login UI

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: web/react/components/signup_team_complete.jsx [hunk 1]
```diff
@@ -42,11 +42,15 @@ WelcomePage = React.createClass({
             state.email_error = "";
         }
 
-        client.signupTeam(email, this.props.state.team.name,
+        client.signupTeam(email,
```
Comment (rgarmsen2295) [inline web/react/components/signup_team_complete.jsx]: Should this be team.email?

File: web/react/components/signup_team_complete.jsx [hunk 2]
```diff
@@ -461,14 +480,16 @@ SendInivtesPage = React.createClass({
...
-                <img className="signup-team-logo" src="/static/images/logo.png" />
-                <h2>Send Invitations</h2>
-                { emails }
-                <div className="form-group"><button type="button" className="btn-default btn" onClick={this.submitAddInvite}>Add Invitation</button></div>
-                <div className="form btn-default-group"><button type="button" className="btn btn-default" onClick={this.submitBack}><i className="glyphicon glyphicon-chevron-left"></i> Back</button>&nbsp;<button type="submit" className="btn-primary btn" onClick={this.submitNext}>Next<i className="glyphicon glyphicon-chevron-right"></i></button></div>
-            </form>
-                <p>{"If you'd prefer, you can send invitations after you finish setting up the "+ strings.Team + "."}</p>
-                <div><a href="#" onClick={this.submitSkip}>Skip this step</a></div>
+                    <img className="signup-team-logo" src="/static/images/logo.png" />
+                    <h2>Invite Team Members</h2>
```
Comment (rgarmsen2295) [inline web/react/components/signup_team_complete.jsx]: Should this be something like "Invite " + strings.Team + " Members"?

File: web/react/components/signup_team_complete.jsx [hunk 3]
```diff
@@ -461,14 +480,16 @@ SendInivtesPage = React.createClass({
...
+                    { emails }
+                    <div className="form-group text-right"><a href="#" onClick={this.submitAddInvite}>Add Invitation</a></div>
+                    <div className="form-group"><button type="submit" className="btn-primary btn" onClick={this.submitNext}>Next<i className="glyphicon glyphicon-chevron-right"></i></button></div>
+                </form>
+                <p className="color--light">if you prefer, you can invite team members later<br /> and <a href="#" onClick={this.submitSkip}>skip this step</a> for now.</p>
```
Comment (rgarmsen2295) [inline web/react/components/signup_team_complete.jsx]: strings.Team here as well?

File: web/react/components/signup_team_complete.jsx [hunk 4]
```diff
@@ -512,19 +533,24 @@ UsernamePage = React.createClass({
             <div>
                 <form>
                 <img className="signup-team-logo" src="/static/images/logo.png" />
-                <h2>Choose a username</h2>
-                <div className={ name_error ? "form-group has-error" : "form-group" }>
-                <div className="row">
-                    <div className="col-sm-9">
-                        <input autoFocus={true} type="text" ref="name" className="form-control" placeholder="" defaultValue={this.props.state.user.username} maxLength="128" />
+                <h2 className="margin--less">Your username</h2>
+                <h5 className="color--light">Select a memorable username that makes it easy for teammates to identify you:</h5>
```
Comment (rgarmsen2295) [inline web/react/components/signup_team_complete.jsx]: Here too

**Comment by crspeller** [comment]
Looks good to me.

File: web/react/components/signup_team_complete.jsx (hunk 1 above)
Comment (jwilander) [inline web/react/components/signup_team_complete.jsx]: Nope, if you scroll up a bit in the file you can see that this is for a new email over the one that the user originally chose

Comment (rgarmsen2295) [inline web/react/components/signup_team_complete.jsx]: Ah ok cool, yup sounds good.

**Comment by coreyhulen** [comment]
besides reeds strings.Team comments +1

**Pull request merged**